examples/notebook/index.css

```diff
@@ -11,10 +11,12 @@ body {
 
 #main {
   position: absolute;
-  top: 4px;
-  left: 4px;
-  right: 4px;
-  bottom: 4px;
+  top: 24px;
+  bottom: 24px;
+  left: 15%;
```

Comment:
I'm not sure what the fix is just yet, but it appears that this `15%` positioning has interfered with the logic around placing the completer. Now, when invoking the completer it places it in the wrong position (at least on my machine):
![image](https://user-images.githubusercontent.com/5728311/50798212-7938d580-128c-11e9-8760-4b9228141275.png)

Reply:
The relevant logic in the completer is here: https://github.com/jupyterlab/jupyterlab/blob/1757615a371c781836ef96c223a3c5f3bc76808e/packages/completer/src/widget.ts#L458-L474

Reply:
Right. I'm not really sure what to do about it though. Maybe I should set width in a different way?

Reply:
I'm not sure either right now. It would be good if the completer could work this out by itself, since we would like it to work in non-absolutely positioned elements. Perhaps @afshin has an idea?

Reply:
I'm also happy to take that styling CSS change out of this PR and move it elsewhere.

Reply:
@yuvipanda @ian-r-rose Maybe we should take out the CSS for this PR and then we use that CSS to open an issue re: the completer so that we aren't blocked by a completer bug.

Reply:
@yuvipanda I've figured out a fix for the completer positioning, and pushed it to your branch as 
d67ce28

Reply:
That's super awesome, @ian-r-rose! Thank you very much.